src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -99,11 +99,11 @@ public ISymbolNode ReadyToRunHelperFromTypeLookup(ReadyToRunHelperId id, Object
             return _genericReadyToRunHelpersFromType.GetOrAdd(new ReadyToRunGenericHelperKey(id, target, dictionaryOwner));
         }
 
-        private NodeCache<ReadOnlyDataBlobKey, BlobNode> _readOnlyDataBlobs;
+        private NodeCache<SettableReadOnlyDataBlobKey, SettableReadOnlyDataBlob> _readOnlyDataBlobs;
 
-        public BlobNode ReadOnlyDataBlob(Utf8String name, byte[] blobData, int alignment)
+        public SettableReadOnlyDataBlob ReadOnlyDataBlob(Utf8String name)
```

Comment:
Can you rename the method to `SettableReadOnlyDataBlob`?

Reply:
Renamed. Thank you!